Integrations/integration-SplunkPy.yml

```diff
@@ -316,7 +316,16 @@ script:
         ec['Splunk.Result'] = res
         if len(dbot_scores) > 0:
             ec['DBotScore'] = dbot_scores
-        demisto.results({"Type": 1, "ContentsFormat": "json", "Contents": json.dumps(res), "EntryContext": ec})
+
+        headers=""
+        if (res and len(res)>0):
+            if not isinstance(res[0], dict):
+                headers = "results"
+
+        humanReadable = tableToMarkdown("Splunk Search results for: " + demisto.args()['query'], res, headers)
+
+        demisto.results({"Type": 1, "Contents": res, "ContentsFormat": "json", "EntryContext": ec, "HumanReadable": humanReadable})
```

Comment:
format this dictionary according to https://github.com/demisto/content/tree/master/docs/code_conventions

Reply:
`humanReadable` =>  `human_readable` var is not according to the conventions
https://github.com/demisto/content/tree/master/docs/code_conventions